src/NzbDrone.Core/HealthCheck/Checks/ImportListRootFolderCheck.cs

```diff
@@ -40,7 +41,7 @@ public override HealthCheck Check()
                     continue;
                 }
 
-                if (!_diskProvider.FolderExists(rootFolderPath))
+                if (rootFolderPath.IsNullOrWhiteSpace() || !_diskProvider.FolderExists(rootFolderPath))
```

Comment:
How would `rootFolderPath` be null or empty? Are we missing validation elsewhere?

If it's null or empty then we're going to add an empty key to the dictionary and tell the user the root folder is missing, instead of telling them it wasn't set.

Reply:
Marking the conversation as resolved doesn't answer the question.